pilot/pkg/model/service.go

```diff
@@ -166,6 +166,10 @@ type Port struct {
 	// service.
 	Port int `json:"port"`
 
+	// TargetPort is the default port where the instances behind this Service can be reached.
```

Comment:
honestly not sure this is a good idea. I hacked this together to get things working, but we really need a more holistic approach to this cross registry stuff to support this better